### ISSUE TITLE
[Bug]: Automation broken

### ISSUE THREAD

**Issue by itsdinkd** [MattiDragon/ExtendedDrawers#1]
### Description of the issue

Hello, i'm the creator of Another Quality Modpack. I'm loving your mod, and I am releasing it in my 1.18.2 release of AQM2. 

I did find a issue however and that's with automation, well so far I've only tried hoppers into the access point. 

It looks like its not finding the item that is already in the drawer, but instead, putting that same item in a new empty slot. When there are no more empty slots, the hopper is storing the apples.

![image](https://user-images.githubusercontent.com/35115304/166121898-f21414c3-d978-4ef7-8b29-894714c0dbdf.png)

### Reproduction steps

1. Build drawers and connect with connector and access point
2. Hopper on top of access point
3. Throw in apples into hopper
4. It will fill empty drawers and not on the drawer that already has the item, and once the slots are full (even with just 1 apple in each drawer in this case) it will no longer take the apple, and will hold the apples in the hopper

### Mod version

1.0.2

### Operating System

w10

### Minecraft version

1.18.2

### Other mods

51.3 fapi

### Server

Not tested

### Client

Happens

### Logs

no error

**Comment by MattiDragon** [comment]
I must have broken the search algorithm at some point. Will look into it later today.

**Comment by MattiDragon** [comment]
I can't seem to reproduce it. Are you sure some other mod isn't affecting it? Maybe something that tweaks hoppers?

**Comment by itsdinkd** [comment]
> I can't seem to reproduce it. Are you sure some other mod isn't affecting it? Maybe something that tweaks hoppers?

Only thing I can honestly think of is Lithium

**Comment by MattiDragon** [comment]
Did you test it with only ED and fabric api? The idea is that you report all mods that are necessary to reproduce the bug.

**Comment by MattiDragon** [comment]
Just tested with lithium. No problems

**Comment by itsdinkd** [comment]
> Did you test it with only ED and fabric api? The idea is that you report all mods that are necessary to reproduce the bug.

Well, knowing it isnt extended drawers itself i unfortunately I do not have the time to test with only necessary mods. I have 2 kids, a full time career, and 3 modpacks and 5 servers to manage, fml. Just figured I'd make you aware of what I encountered when I added the mod into my pack. Wish I could have more time to figure out the conflicting mod, and understand completely you not being able to move forward until I do figure out which one. 

I will close the issue until I can do what you requested.
